cli/tests/services.bats

```diff
@@ -783,9 +783,7 @@ EOF
   # Kill sleep for now just to be safe.
 
   run "$FLOX_BIN" activate -s -- true
-  assert_line "❌ Failed to start services:"
-  # Outputs process-compose log file on failure:
-  assert_output --partial "listening /no_permission.sock"
+  assert_output --partial "❌ Failed to start services"
```

Comment:
I force pushed to replace the `assert_line` with `assert_output --partial` so that we can conditionally match on the trailing colon.